packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md

```diff
@@ -4,7 +4,20 @@ title: Okta Hosted Vs. Customer Hosted
 
 ## Okta Hosted Vs. Customer Hosted
 
-Okta offers multiple integration options when designing an authentication experience. The two most recommended concepts are using the Okta-hosted Sign-In Widget and the Customer-hosted Sign-In Widget. Both of these Okta-built experiences provide easy implementation and maintenance, industry best practice, user self service, and built-in user registration. The difference between the two concepts is where the Sign-In Widget itself is hosted.
+Okta offers multiple integration options when designing an authentication experience. The two most recommended concepts are using the Okta-hosted Sign-In Widget and the Customer-hosted Sign-In Widget. Both of these Okta-built experiences provide easy implementation and maintenance, industry best practice, user self service, and built-in user registration. You can customize the Okta URL domain and the Okta-hosted or Customer-hosted Sign-In Widget style. See [Customize the Okta URL domain](/docs/guides/custom-url-domain/overview/) and [Style the Widget](/docs/guides/style-the-widget/before-you-begin/).
+
+The difference between the two concepts is where the Sign-In Widget itself is hosted. See [Okta-hosted widget](/docs/concepts/hosted-vs-embedded/#okta-hosted-widget) and [Customer-hosted widget](/docs/concepts/hosted-vs-embedded/#customer-hosted-widget) for diagrams on where the Sign-In Widget is hosted.
```

Comment:
```suggestion
The difference between the two integration options is where the Sign-In Widget is hosted. See [Okta-hosted widget](#okta-hosted-widget) and [Customer-hosted widget](#customer-hosted-widget) for hosted scenarios.
```

Reply:
I didn't see diagrams in the topic. If the diagrams are to be added, then please revert this suggestion. Thanks

Reply:
The diagrams are in the sections as referenced.